### ISSUE TITLE
Support keyboard navigation for the Photo Gallery Grid display

### ISSUE THREAD

**Issue by bberndt-uaz** [az-digital/az_quickstart#4820]
## Motivation
The Grid display of the Photo Gallery component is not accessible by keyboard.

## Proposed Resolution
Arizona Bootstrap has been updated to provide this support:
 - https://github.com/az-digital/arizona-bootstrap/pull/1731

The markup for the Photo Gallery Grid display should be updated in Quickstart to go along with the changes from Arizona Bootstrap.
